cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc

```diff
@@ -49,10 +49,12 @@ $ mkdir -p ${SCRATCH}
 $ echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
 ----
 
-[id="rosa-mobb-cloudwatch-sts-prep-aws"]
-== Prepare AWS Account
+. Ensure all fields output correctly before moving to the next section.
```

Comment:
What if you make this an action command. Like:

```
. Ensure all exported environmental variables output correctly before moving to the next section by running the following command:
+
[source,terminal]
----
$ $ echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
----
```

Then, remove that command from the previous box. The `export` commands set these variables, while the `echo` command prints their results.